GliaWidgets/Configuration.swift

```diff
@@ -7,26 +9,35 @@ public enum Environment {
     /// Beta environment. For development use.
     case beta
 
-    var url: String {
+    var region: Salemove.Region {
         switch self {
-        case .europe:
-            return "https://api.salemove.eu"
         case .usa:
-            return "https://api.salemove.com"
+            return .us
+        case .europe:
+            return .eu
         case .beta:
-            return "https://api.beta.salemove.com/"
+            return .custom(URL(string: "https://api.beta.salemove.com/")!)
         }
     }
 }
 
 /// Glia's engagement configuration.
 public struct Configuration {
     /// Application token
-    public let appToken: String
+    @available(*, deprecated, message: "Use `authorizationMethod` instead.")
+    public var appToken: String {
+        if case .appToken(let value) = authorizationMethod {
+            return value
+        } else {
+            return ""
+        }
+    }
     /// Deprecated.
     /// The current provided api token
     @available(*, deprecated, message: "Api token is not supported.")
     public let apiToken: String = ""
+
+    public let authorizationMethod: AuthorizationMethod
```

Comment:
I suppose this needs to have documentation as well.

Reply:
fixed - 7e6540f52dd9be35c623a4f2b0b34c922e67d49d